docs/docs/08-guides/01-terraform-backend.md

```diff
@@ -98,7 +98,7 @@ Replace `<bucket-name>`, `<region>`, `path/to/my/key/terraform.tfstate` and `<ta
 
 
 ```bash
-wing compile -t tf-aws --plugins=plugin.static-backend.js main.w
+wing compile -t tf-aws -t plugin.static-backend.js main.w
```

Comment:
The word "plugin" is still used in this doc, do we want to change that as well?

Reply:
Yea its probably confusing, let me update